packages/core/metadata.typelib.ts

```diff
@@ -0,0 +1,24 @@
+/*
+* This file is generated automatically by the RTTIST TypeGen tool.
```

Comment:
Is this generated by typegen.ts ?

Reply:
Generated by CLI `typegen generate`.
Typegen.ts does not generate anything in the end... With new version of @rttist/typegen I think I can delete that script. That's why I keep it as draft. I'd like to clean it, but I had no time to do so.

BTW I think I put that CLI cmd to package.json scripts.

Reply:
@rttist/typegen updated
scripts/typegen.ts removed